docs/.vitepress/config/sidebar/main.ts

```diff
@@ -6,6 +6,7 @@ export const mainSidebar: DefaultTheme.Sidebar = {
       text: 'Guide',
       collapsible: true,
       items:[
+        { text: "What's envd?", link: '/guide/intro' },
         { text: 'Getting Started', link: '/guide/getting-started' },
```

Comment:
Click "Get Started" on the landing page will go to the 2nd page, which is a little weird.